crates/bevy_ecs/src/system/mod.rs

```diff
@@ -14,3 +14,7 @@ pub use query::*;
 pub use system::*;
 pub use system_chaining::*;
 pub use system_param::*;
+
+// Electric boogaloo
```

Comment:
:laughing:

Reply:
It was easier to manage things creating two concurrent versions than to modify the existing one in-place.

Reply:
Makes sense to me. I just like fun comments like this :)